android/java/org/chromium/chrome/browser/toolbar/top/BraveToolbarLayoutImpl.java

```diff
@@ -695,19 +696,19 @@ private void addMediaToPlaylist(PlaylistItem[] items) {
                             playlistItems.add(playlistItem);
                         }
                     }
-                    if (playlistItems.size() > 0) {
-                        mPlaylistService.addMediaFiles(
-                                playlistItems.toArray(new PlaylistItem[0]),
-                                ConstantUtils.DEFAULT_PLAYLIST,
-                                true,
-                                addedItems -> {
-                                    if (addedItems.length > 0) {
-                                        showAddedToPlaylistSnackBar();
-                                    }
-                                });
-                    } else {
-                        showAlreadyAddedToPlaylistSnackBar();
-                    }
+                    // if (playlistItems.size() > 0) {
+                    mPlaylistService.addMediaFiles(
+                            playlistItems.toArray(new PlaylistItem[0]),
+                            ConstantUtils.DEFAULT_PLAYLIST,
+                            true,
+                            addedItems -> {
+                                if (addedItems.length > 0) {
+                                    showAddedToPlaylistSnackBar();
+                                }
+                            });
+                    // } else {
+                    //     showAlreadyAddedToPlaylistSnackBar();
```

Comment:
do we still need that comment?

Reply:
we need to add it probably later as we are changing things. removing for now.